src/core_plugins/kibana/common/tutorials/filebeat_instructions.js

```diff
@@ -17,93 +17,147 @@
-        '({config.docs.beats.filebeat}/filebeat-getting-started.html).',
+      title: i18n.translate('kbn.common.tutorials.filebeatInstructions.install.rpmTitle', {
+        defaultMessage: 'Download and install Filebeat',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.filebeatInstructions.install.rpmTextPre', {
+        defaultMessage: 'First time using Filebeat? See the [Getting Started Guide]({linkUrl}).',
+        values: {
+          linkUrl: '{config.docs.beats.filebeat}/filebeat-getting-started.html',
+        },
+      }),
       commands: [
         'curl -L -O https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-{config.kibana.version}-x86_64.rpm',
         'sudo rpm -vi filebeat-{config.kibana.version}-x86_64.rpm',
       ],
-      textPost:
-        'Looking for the 32-bit packages? See the [Download page](https://www.elastic.co/downloads/beats/filebeat).',
+      textPost: i18n.translate('kbn.common.tutorials.filebeatInstructions.install.rpmTextPost', {
+        defaultMessage: 'Looking for the 32-bit packages? See the [Download page]({linkUrl}).',
+        values: {
+          linkUrl: 'https://www.elastic.co/downloads/beats/filebeat',
+        },
+      }),
     },
     WINDOWS: {
-      title: 'Download and install Filebeat',
-      textPre:
-        'First time using Filebeat? See the [Getting Started Guide]' +
-        '({config.docs.beats.filebeat}/filebeat-getting-started.html).\n' +
-        '1. Download the Filebeat Windows zip file from the [Download](https://www.elastic.co/downloads/beats/filebeat) page.\n' +
-        '2. Extract the contents of the zip file into `C:\\Program Files`.\n' +
-        '3. Rename the `filebeat-{config.kibana.version}-windows` directory to `Filebeat`.\n' +
-        '4. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select' +
-        ' **Run As Administrator**). If you are running Windows XP, you might need to download and install PowerShell.\n' +
-        '5. From the PowerShell prompt, run the following commands to install Filebeat as a Windows service.',
+      title: i18n.translate('kbn.common.tutorials.filebeatInstructions.install.windowsTitle', {
+        defaultMessage: 'Download and install Filebeat',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.filebeatInstructions.install.windowsTextPre', {
+        // eslint-disable-next-line no-multi-str
+        defaultMessage: 'First time using Filebeat? See the [Getting Started Guide]({guideLinkUrl}).\n\
+ 1. Download the Filebeat Windows zip file from the [Download]({filebeatLinkUrl}) page.\n\
+ 2. Extract the contents of the zip file into {folderPath}.\n\
+ 3. Rename the `{directoryName}` directory to `Filebeat`.\n\
+ 4. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select\
+**Run As Administrator**). If you are running Windows XP, you might need to download and install PowerShell.\n\
```

Comment:
'... and select**Run As Administrator**) ...'
Whitespace shouldn't be removed here